src/main/java/org/mitre/synthea/export/CSVExporter.java

```diff
@@ -709,8 +712,9 @@ private void condition(String personID, String encounterID, Entry condition) thr
    * @param allergy     The allergy itself
    * @throws IOException if any IO error occurs
    */
-  private void allergy(String personID, String encounterID, Entry allergy) throws IOException {
-    // START,STOP,PATIENT,ENCOUNTER,CODE,DESCRIPTION
+  private void allergy(String personID, String encounterID, HealthRecord.Allergy allergy)
+      throws IOException {
+    // START,STOP,PATIENT,ENCOUNTER,CODE,DESCRIPTION,REACTION1,SEVERITY1,REACTION2,SEVERITY2
```

Comment:
Looking at the output here, I'm wondering if we want to add a couple more columns.
Sample: [allergies.csv](https://github.com/synthetichealth/synthea/files/6678076/allergies.csv)

- Either code system or type would be useful here, to know whether the allergy is to a medication or something else. A person can look at it and know penicillin is a medication so it's an RxNorm code, pet dander is a SNOMED code, but for anything automated it would be nice to be explicit.
- The reactions at the end are just codes with no description. I'd be fine with keeping just the code, since it's consistently SNOMED and can be looked up, but just about everywhere else in the CSV exporter we include the description with the code. (And there's at least one place we use the description without the code)